libs/blocks/reading-time/reading-time.css

```diff
@@ -12,3 +14,52 @@ main > .section > .content.reading-time > span {
 .center .reading-time > span {
   text-align: center;
 }
+
+main > .section > .content.inline-wrapper {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 8px 0 12px;
+}
```

Comment:
There is a duplicate of this in the [Share css](https://github.com/adobecom/milo/pull/467/files#diff-f5946602c5148d0f9a4f0705c1e71bcc2446191a670372af4fe68c519d494c8dR139-R144 ). Is it needed in both files?

Reply:
I figured I would put it in both blocks, in case one block is loaded but not the other or vice versa

Reply:
Only `reading-time (inline)` loaded: https://main--stock--adobecom.hlx.page/drafts/solene/reading-time-inline?milolibs=share-block-updates
Only `share (inline)` loaded: https://main--stock--adobecom.hlx.page/drafts/solene/share-inline?milolibs=share-block-updates

Reply:
Hmm. What are we gaining by only having one block set to the `inline` variant? 

The real magic happens when two blocks are set to `inline` and they sit in the same row. Which, means there are  unnecessary [DOM updates](https://github.com/adobecom/milo/pull/467/files/fbb582fc882c7a9ce5b784501bc0328f1938c2b2#diff-4995aeeb074dd406bdcb22a5460207f2228dda32b90545b35f1df684658e6760R34) happening if only one block is set to `inline`.  I think you can look for the existence of a sibling element with the `inline` variant and only update the DOM if there is.

Reply:
Good point! It's now fixed so that it won't affect the DOM if it's a single sibling.